apps/web-client/src/lib/forms/DaisyUIBookForm.svelte

```diff
@@ -0,0 +1,90 @@
+<script lang="ts">
+	import type { FormOptions, SuperValidated, SuperForm } from "sveltekit-superforms";
+	import { superForm, numberProxy, stringProxy } from "sveltekit-superforms/client";
+
+	import { testId } from "@librocco/shared";
+
+	import type { BookFormSchema } from "$lib/forms/schemas";
+
+	export let data: SuperValidated<BookFormSchema>;
+	export let options: FormOptions<BookFormSchema>;
+	export let publisherList: string[] = [];
+
+	export let isExtensionAvailable: boolean;
+
+	/**
+	 * Handle click of "X" icon button
+	 */
+	export let onCancel: (e: Event) => void = () => {};
+
+	export let onFetch: (isbn: string, form: SuperForm<BookFormSchema>["form"]) => void = () => {};
+
+	const form = superForm(data, options);
+
+	const { form: formStore, constraints, enhance } = form;
+
+	const priceProxy = numberProxy(formStore, "price", { empty: "undefined" });
+	const publisherProxy = stringProxy(formStore, "publisher", { empty: "undefined" });
+</script>
+
+<form
+	class="divide-y-gray-50 flex h-auto flex-col gap-y-6 px-4 py-4"
+	aria-label="Edit book details"
+	use:enhance
+	method="POST"
+	id={testId("book-form")}
+>
+	<div class="flex justify-between">
+		<label class="input-bordered input flex items-center gap-2">
+			ISBN
+			<input bind:value={$formStore.isbn} name="isbn" type="text" class="grow" placeholder="0000000000" {...$constraints.isbn} disabled />
+		</label>
+		<button disabled={!isExtensionAvailable} type="button" class="btn-secondary btn" on:click={() => onFetch($formStore.isbn, formStore)}>
```

Comment:
This button really needs to provide more feedback about the state of the request. The original form didn't have it either, so I don't think it needs to be tackled here. Just a note that I've opened #778